x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/mapper/ScriptFieldMapper.java

```diff
@@ -131,6 +125,27 @@ public ScriptFieldMapper build(BuilderContext context) {
             );
         }
 
+        private MappedFieldType buildType(String fullName) {
+            switch (runtimeType.getValue()) {
```

Comment:
I was making changes to the lookup as well in #59762 , let's use the constants from the existing mappers for their type?

Reply:
I'll use your lookup, sure.